src/CommandHandler.cpp

```diff
@@ -106,8 +106,15 @@ void CommandHandler::init(std::string ctrls)
     std::string item;
 
     while (std::getline(ss, item, XENCAMERA_LIST_SEPARATOR[0])) {
-        LOG(mLog, DEBUG) << "Assigned control: " << item;
-        mControls.push_back(item);
+        /*
+         * As missing "controls" string in domain config results in a "(null)"
+         * string in corresponding xenstore entry, treat this invalid string as
+         * an empty string and do not try to assign it.
+         */
+        if (item != "(null)" ) {
```

Comment:
This could probably be fixed at the origin, e.g. Xen tool stack

Reply:
agree, will fix in tool stack